packages/playground/src/components/Header.tsx

```diff
@@ -0,0 +1,194 @@
+import { useCallback } from 'react';
+
+import Form, { IChangeEvent } from '@rjsf/core';
+import { RJSFSchema, UiSchema, ValidatorType } from '@rjsf/utils';
+import localValidator from '@rjsf/validator-ajv8';
+
+import CopyLink from './CopyLink';
+import ThemeSelector, { ThemesType } from './ThemeSelector';
+import Selector from './Selector';
+import ValidatorSelector from './ValidatorSelector';
+import SubthemeSelector from './SubthemeSelector';
+import RawValidatorTest from './RawValidatorTest';
+import { LiveSettings } from './Playground';
```

Comment:
Rather than importing `LiveSettings` export it from here instead. It prevents a circular import reference between `Playground` and `Header`
```suggestion

export interface LiveSettings {
  showErrorList: false | 'top' | 'bottom';
  [key: string]: any;
}
```